vermouth/selectors.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from .molecule import Molecule
+
+
+PROTEIN_RESIDUES = ('ALA', 'ARG', 'ASP', 'ASN', 'CYS',
+                    'GLU', 'GLN', 'GLY', 'HIS', 'ILE',
+                    'LEU', 'LYS', 'MET', 'PHE', 'PRO',
+                    'SER', 'THR', 'TRP', 'TYR', 'VAL')
```

Comment:
Do we want to move this to a FF specific datafile? I think we do.
Either way, make it a set instead of a tuple.

Reply:
It is my plan indeed. But I would like to add arbitrary attribute to force fields, and I have some thinking to do about the API. I'll change that in an other PR if you don't mind. I made it a set, though.